### PR TITLE
Enable multiple anchors and to top single anchor

### LINKED ISSUE #11: Update anchor CSS to fit with Aqua theme

### LINKED ISSUE #18: Apply multiple anchors to homepage

### PR DESCRIPTION
**Pull request by R-Jim**
Closes #18 
Preview
Multiple anchors section
image
on hover
image
To top single anchor button
image

### CODE REVIEW

**Comment by zapplecat** [comment]
I can't really see the TOP button when I'm all the way down.
image

Maybe forgo the TOP button and add a top section to the multi-anchor?

File: frontend/src/assets/icons/anchorIcon.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<svg width="40px" height="40px" viewBox="0 0 60 60" xmlns="http://www.w3.org/2000/svg">
+ <style>.stb0{stroke:#fff} .stf0{fill:#fff}</style>
+ <g>
+  <title>Layer 1</title>
```
Comment (zapplecat) [inline frontend/src/assets/icons/anchorIcon.svg]: This is showing up as a alt-text when hovering over the anchor. I'd remove or change it to `page-anchor`. Better yet, to support accessibility, we should have the alt-text that have the name of the anchored section.

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -47,12 +50,16 @@ export default class App extends React.Component<AppProps, LanguageContextValue>
...
+                        <Route path='/home' component={HomePage} />
                     </Switch>
                 </main>
-                <div style={{height: "25px"}}/>
+                <div style={{ height: "25px" }} />
                 <AltNav />
                 <FooterSection />
+                <AnchorSingleSection position={AnchorSectionPosition.BOTTOM_RIGHT} anchor={{
+                    href: "#header",
+                    svgIcon: AnchorBotan, //TODO: replace this with aqua onion
```
Comment (zapplecat) [inline frontend/src/App.tsx:61]: Just leaving a comment here to make sure we don't forget to replace the icon. Not sure if we want to merge before/after.

File: frontend/src/components/comboSection/comboSection.tsx [hunk 3]
```diff
@@ -25,26 +25,26 @@ export default class ComboSection extends BaseSection<Message | Artwork | Video
         // TODO: messagecard-center might not used or needed
         if ("messageID" in object) {
             return (
-                <div className="card-section">
+                <div key={id} className="card-section">
```
Comment (zapplecat) [inline frontend/src/components/comboSection/comboSection.tsx:28]: Thanks for fixing these. I kept seeing the console error but smol brain hasn't processed it was coming from here.

File: frontend/src/pages/home/home.tsx [hunk 4]
```diff
@@ -31,8 +35,27 @@ export interface HomePageState {
...
+const Anchors: Anchor[] = [
+    {
+        href: "#video-anchor",
+        svgIcon: AnchorBotan,
+        text: "Video",
+    },
+    {
+        href: "#message-anchor",
+        svgIcon: AnchorBotan,
+        text: "Message",
```
Comment (zapplecat) [inline frontend/src/pages/home/home.tsx]: nit: `Messages` instead of message.

**Review by zapplecat — APPROVED** [review]
Left some minor comments, overall lgtm

**Comment by DuongTyler** [comment]
I was thinking we could make it fade in and out transparently using :hover in the CSS. I think it might look a little better that way

**Comment by DuongTyler** [comment]
Also, it doesn't look good on mobile phones. We might want to consider making a special rule to hide it on phone layouts in the CSS

**Comment by DuongTyler** [comment]
image
I didn't notice that this was a thing, perhaps we should make it look more like a button with a circular colored background instead? also I didn't exactly know what clicking this would do.

**Review by DuongTyler — CHANGES_REQUESTED** [review]
see above comments

**Comment by R-Jim** [comment]
> image
> I didn't notice that this was a thing, perhaps we should make it look more like a button with a circular colored background instead? also I didn't exactly know what clicking this would do.

I just left it as a placeholder until we get the icon for it but yeah I will update it

**Comment by R-Jim** [comment]
> see above comments

All are updated in description

**Comment by DuongTyler** [comment]
I think it might be better to have the visibility of the "jump to top" button also be transparent until hovered over.
Also we might as well just some material design arrow or button instead. Makes things simple to understand the function of

**Comment by DuongTyler** [comment]
image
also Minor nit: sometimes the position of the sidebar doesn't update on the page

**Review by DuongTyler — APPROVED** [review]
lgtm, but I would change the anchor icon to an up arrow or something

**Comment by DuongTyler** [comment]
by that I mean the jump to top button

**Comment by zapplecat** [comment]
> lgtm, but I would change the anchor icon to an up arrow or something

lol, do a reverse anchor to make it look like it's pointing it. (But really, don't, it would probably look terrible.)

**Comment by R-Jim** [comment]
> > lgtm, but I would change the anchor icon to an up arrow or something
> 
> lol, do a reverse anchor to make it look like it's pointing it. (But really, don't, it would probably look terrible.)

lol I will create a new svg icon for it

**Pull request merged**